src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.scss

```diff
@@ -6,6 +6,13 @@
     color: $bdl-box-blue;
     font-weight: bold;
 
+    &::before {
```

Comment:
I think this change makes sense, but can we move it back to the `AnnotationActivity` stylesheet? In theory, the AnnotationActivityLink component could be reused elsewhere and may have a different layout/experience.